test/bin/missing-dune-project/dune

```diff
@@ -0,0 +1,2 @@
+(cram
+ (deps %{bin:opam-monorepo}))
```

Comment:
If you move this file up and add `(applies_to :whole_subtree)` it'll apply to all future cram tests.

Reply:
Good point! I'll move it!